src/clarity-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -16,18 +20,49 @@ let nbTabLinkComponents: number = 0;
         "[attr.aria-controls]" : "ariaControls",
         "role": "presentation",
         "[class.btn]": "true",
-        "[class.btn-link]": "true",
-        "[class.nav-link]": "true",
-        "[class.nav-item]": "true",
+        "[class.btn-link]": "!inOverflow",
+        "[class.nav-link]": "!inOverflow",
+        "[class.nav-item]": "!inOverflow",
         "[class.active]": "active"
     }
 })
 export class TabLinkDirective {
-    @Input("clrTabLinkId") tabLinkId: string;
-    ariaControls: string;
 
-    constructor(public ifActiveService: IfActiveService, @Inject(IF_ACTIVE_ID) private id: number) {
+    @Input("clrTabLinkInOverflow") inOverflow: boolean;
+    templateRefContainer: TemplateRefContainer;
+
+    constructor(public ifActiveService: IfActiveService, @Inject(IF_ACTIVE_ID) private id: number,
+                private  ariaService: AriaService, private el: ElementRef,
```

Comment:
Is this private because its returned from `(public) ariaControls` but not used directly in the view? I noticed that it's public in `tab-content.ts`